aea/crypto/ledger_apis.py

```diff
@@ -99,6 +100,31 @@ def transfer(self, identifier: str, entity: Entity, destination_address: str, am
             tx_digest = None
         return tx_digest
 
+    def is_tx_settled(self, identifier: str, tx_digest: str, amount: float) -> bool:
+        """
+        Check whether the transaction is settled and correct.
+
+        :param identifier: the identifier of the ledger
+        :param tx_digest: the transaction digest
+        :param amount: the amount
+        :return: True if correctly settled, False otherwise
+        """
+        assert identifier in self.apis.keys(), "Unsupported ledger identifier."
+        is_successful = False
+        try:
+            api = self.apis[identifier]
+            logger.info("Checking the transaction ...")
+            # tx_status = cast(TxStatus, api.tx.status(tx_digest))
+            tx_status = cast(str, api.tx.status(tx_digest))
+            SUCCESSFUL_TERMINAL_STATES = ('Executed', 'Submitted')
+            if tx_status in SUCCESSFUL_TERMINAL_STATES:
+                # TODO: check the amount of the transaction is correct
```

Comment:
Currently, the ledger api does not allow for this check. Ed says it will be available in a few days :).